### ISSUE TITLE
Destruction Gadget doesn't take any energy

### ISSUE THREAD

**Issue by MRT-Functions** [Direwolf20-MC/BuildingGadgets#512]
http://89.36.14.87/poze/F2CuXb4Vys.gif

**Comment by The-Code-Monkey** [comment]
@MRT-Functions what version of forge and the mod are you running as there isnt much we can do without these.

**Comment by MajorTuvok** [comment]
And could you check you haven't set the config accidentally so it doesn't use energy or have some charger in your player inventory/linked tonxour player inventory?

**Comment by NillerMedDild** [comment]
I can confirm that none of the gadgets consume energy, but do not work while completely unpowered.

### Versions
- Forge-1.16.1-32.0.108
- buildinggadgets-3.5.0

### Logs
My latest log, because why not. Nothing in there about Building Gadgets erroring though.
https://gist.github.com/NillerMedDild/3e73253b8efca4bcd72ad13d7f7e8ccb

**Comment by MichaelHillcox** [comment]
Could you possibly pastbin your Building Gadgets config files? Client, common and server (server found on the server world or in your worlds serverconfig folder)

**Comment by NillerMedDild** [comment]
Sure, but they're completely default.

**Comment by MichaelHillcox** [comment]
Oh, then don't waste your time, if you've not modifed them and you're not playing on a server then it's very likely to be something else.

**Comment by NillerMedDild** [comment]
client: https://gist.github.com/NillerMedDild/f91e6d181d7f66f41f005cefc5c4715e
server: https://gist.github.com/NillerMedDild/52616759984fa9bd99bfe3ec612a4e8d

I was already at it, no point in stopping :P

**Comment by MichaelHillcox** [comment]
Yeah, completely stock, alrighty, I'll look into it regardless, thanks for the help :+1:
